etc/kayobe/kolla.yml

```diff
@@ -424,6 +444,18 @@ kolla_build_blocks:
     {% endif %}
     {% endraw %}
     {% endif %}
+  nova_compute_footer: |
+    {% if kolla_base_distro in ['centos', 'rocky'] %}
+    # Install networking-mlnx plugin and dependencies
+    {% raw %}
+    {% if base_package_type == 'rpm' %}
+    ADD plugins-archive /
```

Comment:
Package installs look reasonable. I think:

```
  neutron_base_packages_append
    - python3-libvirt 
    - python3-ethtool
  nova_base_packages_append:
    - python3-libvirt
    - python3-ethtool
```

and
```
  neutron_pip_packages_append:
    - /plugins/*
  nova_pip_packages_append:
    - /plugins/*
```

Would be more normal. Also need something similar for ubuntu?

Reply:
Thinking something more like this: https://github.com/stackhpc/stackhpc-kayobe-config/commit/a22bf2deaa1d7cebfbabc3655fd5973bc88f1edb